lib/helper.js

```diff
@@ -5,6 +5,21 @@ const hoek = require('@hapi/hoek');
 const dayjs = require('dayjs');
 const TEMPLATE_NAME_REGEX_WITH_NAMESPACE = schema.config.regex.FULL_TEMPLATE_NAME_WITH_NAMESPACE;
 const formatDate = dateTime => dayjs(dateTime).format('YYYY-MM-DD');
+// SCM handles only SUCCESS, PENDING & FAILURE status
+const SCM_STATE_MAP = {
+    ABORTED: 'FAILURE',
+    CREATED: 'PENDING',
+    FAILURE: 'FAILURE',
+    QUEUED: 'PENDING',
+    RUNNING: 'PENDING',
+    SUCCESS: 'SUCCESS',
+    BLOCKED: 'PENDING',
+    UNSTABLE: 'FAILURE',
+    PENDING: 'PENDING',
```

Comment:
Is PENDING a valid status?
https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L14-L25

Reply:
PENDING is not a valid build status. Changes are done to remove it from this map.